src/main/kotlin/no/nav/familie/ef/sak/behandling/fremleggsoppgave/Fremleggsoppgave.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.familie.ef.sak.behandling.fremleggsoppgave
+
+import org.springframework.data.annotation.Id
+import java.util.UUID
+
+data class Fremleggsoppgave(
+    @Id
+    val behandlingId: UUID,
+    val inntekt: Boolean
```

Comment:
når jeg leser Fremleggsoppgave som domene tenker jeg at det har en kobling til en oppgave, men her er det en kobling til inntekt - som har true/false. Mulig det er noe jeg ikke skjønner, men kan det struktureres på en annen måte?

Reply:
Vi tenkte at en fremleggsoppgave kan være for inntekt - og evt andre typer etterhvert